### ISSUE TITLE
Support OKFN Tabular data package format

### ISSUE THREAD

**Issue by phochste** [LibreCat/Catmandu#175]
http://data.okfn.org/standards

**Comment by nichtich** [comment]
Yes it would be great to have Catmandu listed at http://data.okfn.org/tools. We already support JSON Schema (http://json-schema.org) for validation and [Catmandu::Exporter::Table](https://metacpan.org/pod/Catmandu::Exporter::Table) supports [JSON table schema](http://dataprotocols.org/json-table-schema/) to specify fields. OKFN Tabular data package format consists of three parts:

* CSV files (✓)
* Metadata in JSON (✓) 
* JSON Table Schema (partly ✓)

I think one can already process all of these with Catmandu but processing may be more convenient. A look at existing tools may help to get an idea of use cases.

**Comment by danfowler** [comment]
@phochste @nichtich we have a CSV on GitHub you can edit to add your tool: https://github.com/frictionlessdata/frictionlessdata.io/blob/gh-pages/_data/tools.csv

I'd love to hear more about Catmandu.  Perhaps you can chat about it here: https://gitter.im/frictionlessdata/chat

**Comment by phochste** [comment]
@danfowler Ok will do, and will try to join you on the chat later today
